pkg/services/network/nsxt_vpc_provider.go

```diff
@@ -40,6 +40,11 @@ import (
 	"sigs.k8s.io/cluster-api-provider-vsphere/pkg/services"
 )
 
+const (
+	// AnnotationEnableEndpointHealthCheckKey is the key of the annotation that is used to enable health check on the VMService endpoint port.
+	AnnotationEnableEndpointHealthCheckKey = "lb.iaas.vmware.com/enable-endpoint-health-check"
```

Comment:
```suggestion
	// NSXTVPCEnableEndpointHealthCheckAnnotationKey is the key of the annotation that is used to enable health check on the VMService endpoint port.
	NSXTVPCEnableEndpointHealthCheckAnnotationKey = "lb.iaas.vmware.com/enable-endpoint-health-check"
```

Do we have a link to some code which more describes what this annotation will do? And maybe add some more to the godoc what it does?